ironfish/src/wallet/account.ts

```diff
@@ -23,7 +23,12 @@ import { WalletDB } from './walletdb/walletdb'
 
 export const ACCOUNT_KEY_LENGTH = 32
 
```

Comment:
I think a better update to this would be something like
```
AccountImport = {name: string, spendingKey:string} | {name: string, incomingViewKey: string, outgoingViewKey: string} 
```